experiments/standalone/Bucket/global_bucket_function.jl

```diff
@@ -164,6 +169,38 @@ updatefunc = ClimaLand.make_update_drivers(bucket_atmos, bucket_rad)
 driver_cb = ClimaLand.DriverUpdateCallback(updateat, updatefunc)
 cb = SciMLBase.CallbackSet(driver_cb, saving_cb)
 
+
```

Comment:
This
`saving_cb = ClimaLand.NonInterpSavingCallback(saved_values, saveat);` is no longer required if we use diagnostics

Reply:
From L161 to L168

Reply:
Let's talk about this, 
```
W = [
    Array(Remapping.interpolate(remapper, sol.u[k].bucket.W)) for
    k in 1:length(sol.t)
];
```
returns a `9-element Vector{Matrix{Float64}}`

would be equivalent to ( ? )
```
W = get(simdir; short_name = "wsoil")
W = W.data
```
which returns `23×180×90 Array{Float64, 3}`

(do we need to adjust format a bit, or am I getting the wrong this?)